utils/touchstone-configs/uperf-touchstone-norm.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Why are you adding this config?, norm_ops comparison is already covered by uperf-touchstone.json

Reply:
I didn't mean to add that there, I wanted to keep the uperf-touchstone.json file as is and add in other files that added in other data that used to be reported to help resolve: https://github.com/cloud-bulldozer/e2e-benchmarking/issues/428